src/axom/core/Array.hpp

```diff
@@ -352,6 +352,19 @@ class Array : public ArrayBase<T, DIM, Array<T, DIM, SPACE>>
    */
   void fill(const T& value);
 
+  /*!
+   * \brief Set a range of elements to a given value.
+   *
+   * \param [in] value the value to set to.
+   * \param [in] n the number of elements to write.
+   * \param [in] pos the position at which to begin writing.
+   *
+   * \note The size is unchanged by calls to set.
```

Comment:
```suggestion
   * \note The size is unchanged by calls to fill.
```

Not sure if 'set' is intended here?

Reply:
Agreed but forgot to use.  I'll put in another PR.